libraries/CAN/test/unit/src/framework/arduino/Test_main.ino

```diff
@@ -0,0 +1,64 @@
+// test includes
+#include "Test_includes.hpp"
+
+
+/***
+     * Uses flags / defines to determine which groups of tests should be added to this test such that multiple tests can be executed in a single run.
+     * Use the '--build-property option of the 'arduino_compile' target to add the flags by defining the respective targets in the makefile.
+     * makefile : --build-property "compiler.cpp.extra_flags=\"-D<TEST_SPECIFIER>=1\"" build
+*/
+void RunAllTests(void)
+{
+
+// CAN
+#ifdef TEST_CAN
+
+    RUN_TEST_GROUP(CAN);
+
+#endif
+
+
+// CAN with connections
+#ifdef TEST_CAN_CONNECTED
+
+    RUN_TEST_GROUP(CAN_connected);
+
+#endif
+
+
+// // CAN
```

Comment:
comment on commented parts

Reply:
I removed the Unity source part, but keep the makefile, just easier for me to test before HIL is up and running :D